k8s.gcr.io/ensure-prod-repo.sh

```diff
@@ -0,0 +1,86 @@
+#!/bin/sh
```

Comment:
Out of curiosity, is there a reason you didn't use bash here? Like, I'm fine with `sh` but I saw bash everywhere in kuberenetes/release repo.

Reply:
I try to not say "bash" unless I need bash-isms.